### ISSUE TITLE
Unable to download archived Facebook Live video

### ISSUE THREAD

**Issue by ischmidt20** [ytdl-org/youtube-dl#15859]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.03.10*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [ x] I've **verified** and **I assure** that I'm running youtube-dl **2018.03.10**

### Before submitting an *issue* make sure you have:
- [ x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [ x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [ x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
youtube-dl --username **** --password **** -F "https://www.facebook.com/FOXSportsChampionsLeague/videos/1782104488494402/" --verbose
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--username', 'PRIVATE', '--password', 'PRIVATE', '-F', 'https://www.facebook.com/FOXSportsChampionsLeague/videos/1782104488494402/', '--verbose']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.03.10
[debug] Python version 3.6.2 (CPython) - Darwin-17.4.0-x86_64-i386-64bit
[debug] exe versions: ffmpeg 3.4.1, ffprobe 3.4.1, rtmpdump 2.4
[debug] Proxy map: {}
[facebook] Downloading login page
[facebook] Logging in
[facebook] 1782104488494402: Downloading webpage
[facebook] 1782104488494402: Downloading webpage
ERROR: Cannot parse data; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 785, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.6/site-packages/youtube_dl/extractor/common.py", line 440, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.6/site-packages/youtube_dl/extractor/facebook.py", line 422, in _real_extract
    video_id, fatal_if_no_video=True)
  File "/usr/local/lib/python3.6/site-packages/youtube_dl/extractor/facebook.py", line 337, in _extract_from_url
    raise ExtractorError('Cannot parse data')
youtube_dl.utils.ExtractorError: Cannot parse data; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
...
<end of log>
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://www.youtube.com/watch?v=BaW_jenozKc
- Single video: https://youtu.be/BaW_jenozKc
- Playlist: https://www.youtube.com/playlist?list=PL4lCao7KL_QFVb7Iudeipvc2BCavECqzc

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

I was able to successfully download the previous Facebook Live stream of the first leg a few weeks ago [here](https://www.facebook.com/FOXSportsChampionsLeague/videos/1755266694511515/), although now I am getting the same error. I do have full credentials to view in the browser (and it appears I am able to log in successfully).

**Comment by DaNiSoFt** [comment]
Same problem here with older version and 2018.03.14.
Other videos of that closed facebook group already worked with provided login
some time ago.
The videos are working in browser but youtube-dl fails. Update did not help.
I tried one video that I already downloaded before - it's also not working anymore.

Thanks a lot

**Comment by dbogatev** [comment]
Same problem here with latest version 2018.03.14 - unable to download videos that require login. 
However if no login is provided to youtube_dl videos that do not require login are downloading without issues. 
The problem is only somewhere after the login, as the same videos that are downloading without login fail to download when credentials are added to the youtube_dl command.
Here is the example of video that works without credentials:
```youtube-dl -v -F "https://www.facebook.com/SkylineWebcams/videos/1823658634322275/" --cookie /mnt/upload/youtubedl_cookies.txt
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-F', u'https://www.facebook.com/SkylineWebcams/videos/1823658634322275/', u'--cookie', u'/mnt/upload/youtubedl_cookies.txt']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.03.14
[debug] Python version 2.7.13 (CPython) - Linux-4.9.0-6-amd64-x86_64-with-debian-9.2
[debug] exe versions: ffmpeg 3.2.4, ffprobe 3.2.4
[debug] Proxy map: {}
[facebook] 1823658634322275: Downloading webpage
[facebook] 1823658634322275: Downloading webpage
ERROR: Cannot parse data; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/YoutubeDL.py", line 785, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/common.py", line 440, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/facebook.py", line 422, in _real_extract
    video_id, fatal_if_no_video=True)
  File "/usr/local/lib/python2.7/dist-packages/youtube_dl/extractor/facebook.py", line 337, in _extract_from_url
    raise ExtractorError('Cannot parse data')
ExtractorError: Cannot parse data; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```
And the same without credentials:
```youtube-dl -v -F "https://www.facebook.com/SkylineWebcams/videos/1823658634322275/"
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-F', u'https://www.facebook.com/SkylineWebcams/videos/1823658634322275/']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2018.03.14
[debug] Python version 2.7.13 (CPython) - Linux-4.9.0-6-amd64-x86_64-with-debian-9.2
[debug] exe versions: ffmpeg 3.2.4, ffprobe 3.2.4
[debug] Proxy map: {}
[facebook] 1823658634322275: Downloading webpage
[info] Available formats for 1823658634322275:
format code  extension  resolution note
dash_sd_src  mpd        unknown
dash_hd_src  mpd        unknown    (best)
```

**Comment by chowbok** [comment]
Still not fixed in 2018.03.20. I can paste my output if you want but it looks pretty similar to dbogatev's above.
